metpy/calc/indices.py

```diff
@@ -51,3 +51,48 @@ def precipitable_water(dewpt, p, top=400 * units('hPa')):
     pw = -1. * (np.trapz(w.magnitude, pres_layer.magnitude) * (w.units * pres_layer.units) /
                 (g * rho_l))
     return pw.to('millimeters')
+
+
+@exporter.export
+@check_units('[speed]', '[speed]', '[pressure]', '[length]')
+def mean_wind_pressure_weighted(u, v, p, hgt=None, depth=None, bottom=None):
```

Comment:
`hgt`->`heights`; also let's go with order of `heights`, `bottom`, `depth` for consistency with `get_layer`.